Add linspace test modules

tugsbayasgalan opened the pull request:
Test Plan: none

Differential Revision: D33785383

cccclai reviewed (approved):
Make sure the changes doesn't mix up with https://github.com/pytorch/pytorch/pull/71578 when rebase, and then it's good.

tugsbayasgalan committed:
Add linspace test modules (#71850)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/71850

Test Plan: none

Reviewed By: cccclai

Differential Revision: D33785383

fbshipit-source-id: f6eca12ce6edfac1dae27577a5f7dacfe0c23d6d
